src/main/java/at/makubi/maven/plugin/avrohugger/typeoverride/primitive/AvroScalaTimestampMillisType.java

```diff
@@ -0,0 +1,20 @@
+package at.makubi.maven.plugin.avrohugger.typeoverride.primitive;
+
+
+import avrohugger.types.JavaSqlTimestamp$;
+import avrohugger.types.JavaTimeInstant$;
+
+/**
+ * https://github.com/julianpeeters/avrohugger/blob/master/avrohugger-core/src/main/scala/types/PrimitiveAvroScalaTypes.scala#L14
```

Comment:
I guess the link should be different as well.

Reply:
True, thanks for noticing.

Reply:
Done.